portfolio/src/main/webapp/script.js

```diff
@@ -60,33 +60,55 @@ async function displayComments() {
 }
 
 /**
- * Displays the login status of the user and a link for logging in or out.
+ * Retrieves the login information for the user.
  */
-async function displayLoginInfo() {
+async function getLoginInfo() {
   const response = await fetch('/login');
   const loginInfo = await response.json();
-  const loginUrl = loginInfo[0];
-  const logoutUrl = loginInfo[1];
-  const userEmail = loginInfo[2];
 
+  return loginInfo;
+}
+
+/**
+ * Displays the login status of the user and a link for logging in or out.
+ */
+function displayLoginInfo(loginInfo) {
   const loginContainer = document.getElementById('login-container');
 
   // Check if the user is logged in.
-  if (userEmail === null) {
+  if (!loginInfo.isLoggedIn) {
     loginContainer.innerHTML =
-        '<a href="' + loginUrl + '">Click here to log in</a>';
+        '<a href="' + loginInfo.loginUrl + '">Click here to log in</a>';
   } else {
-    loginContainer.innerHTML = 'Logged in as ' + userEmail + '<br /><a href="' +
-        logoutUrl + '">Click here to log out</a>';
+    loginContainer.innerHTML = 'Logged in as ' + loginInfo.userEmail +
+        '<br /><a href="' + loginInfo.logoutUrl + '">Click here to log out</a>';
+  }
+}
+
+/**
+ * Displays the button for deleting all comments iff the user is an admin.
```

Comment:
"iff" -> "if"

Reply:
Sorry, I was using iff as an abbreviation for "if and only if"